Sources/StructuredQueriesCore/Updates.swift

```diff
@@ -89,3 +89,7 @@ extension Updates: QueryExpression {
     "SET \(updates.map { "\(quote: $0) = \($1)" }.joined(separator: ", "))"
   }
 }
+
+public struct _TableAliasName<Base: Table>: AliasName {
+  public static var aliasName: String { Base.tableName }
+}
```

Comment:
This is the simplest way I could come up with to fix the nested update issue. Underscored type for now since I'm not sure folks should be using it, especially if we come up with a better way to handle things in the future. But if we come up with more use cases for this type, we could consider making a proper public API.